.github/workflows/t3000-demo-tests.yaml

```diff
@@ -1,4 +1,4 @@
-name: "[T3K] T3000 demo tests"
+name: "(T3K) T3000 demo tests"
```

Comment:
is this going to duplicate the workflow listing on Github UI?
What happens to the old workflow with [T3K]?

Reply:
It should be replaced

Reply:
ok, sounds good